src/CodeConverter.jsx

```diff
@@ -59,6 +59,8 @@ const useShikiMonaco = () => {
   return { isReady, highlighter: highlighterRef.current };
 };
 
+import ClipLoader from "react-spinners/ClipLoader";
```

Comment:
import Cliploader, famoon and fasun have already been done at start of this file with other imports.
remove line 62 and line 63